packages/compass-query-bar/src/stores/query-bar-reducer.ts

```diff
@@ -275,8 +275,10 @@ export const explainQuery = (): QueryBarThunkAction<void> => {
     const {
       queryBar: { fields },
     } = getState();
-    const query = mapFormFieldsToQuery(fields);
-    localAppRegistry?.emit('open-explain-plan-modal', { query });
+    const { project, ...query } = mapFormFieldsToQuery(fields);
+    localAppRegistry?.emit('open-explain-plan-modal', {
+      query: { ...query, projection: project },
+    });
```

Comment:
We do the same in crud-store when fecthing docs, so maybe would make sense to eventually rename this to projection in query bar store to match driver interface. Don't want to do this in the same PR though because this would require too many changes int he app for a patch that should be just a bug fix